x-pack/plugins/lists/server/routes/get_exception_filter_route.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { transformError } from '@kbn/securitysolution-es-utils';
+import {
+  CreateExceptionListItemSchema,
+  ExceptionListItemSchema,
+  getExceptionFilterSchema,
+} from '@kbn/securitysolution-io-ts-list-types';
+import { EXCEPTION_FILTER } from '@kbn/securitysolution-list-constants';
+
+import { buildExceptionFilter } from '../services/exception_lists/build_exception_filter';
+import { ListsPluginRouter } from '../types';
+
+import { buildRouteValidation, buildSiemResponse } from './utils';
+
+export const getExceptionFilterRoute = (router: ListsPluginRouter): void => {
+  router.post(
+    {
+      options: {
+        tags: ['access:securitySolution'],
+      },
+      path: `${EXCEPTION_FILTER}`,
+      validate: {
+        body: buildRouteValidation(getExceptionFilterSchema),
+      },
+    },
+    async (context, request, response) => {
+      const siemResponse = buildSiemResponse(response);
+      try {
+        const ctx = await context.resolve(['lists']);
+        const listClient = ctx.lists?.getListClient();
+        if (!listClient) {
+          return siemResponse.error({ body: 'Cannot retrieve list client', statusCode: 500 });
+        }
+        const exceptionListClient = ctx.lists?.getExceptionListClient();
+        const exceptionItems: Array<ExceptionListItemSchema | CreateExceptionListItemSchema> = [];
+        const {
+          type,
+          alias = null,
+          exclude_exceptions: excludeExceptions = true,
+          chunk_size: chunkSize = 1024,
+        } = request.body;
+        if (type === 'exception_list_ids') {
+          for (const {
+            exception_list_id: exceptionListId,
+            namespace_type: namespaceType,
+          } of request.body.exception_list_ids) {
+            const exceptionList = await exceptionListClient?.findExceptionListItem({
```

Comment:
This won't fetch the full exceptions list, just one page of it. Should use `findExceptionListsItemPointInTimeFinder` similar  to the `getExceptions` function that is used in createSecurityRuleTypeWrapper.